src/components/AccountSettings.vue

```diff
@@ -155,6 +164,19 @@ export default {
 		},
 	},
 	methods: {
+		onAccountUpdated() {
```

Comment:
this is a regresssion from another old pr, and fixed it here for time reasons

Reply:
Thanks!